docs/dev/getting-started/initial-setup/managed-edition.md

```diff
@@ -60,18 +60,18 @@ run `composer create-project contao/managed-edition`:
 {
     "scripts": {
         "post-install-cmd": [
-            "Contao\\ManagerBundle\\Composer\\ScriptHandler::initializeApplication"
+            "@php vendor/bin/contao-setup --ansi"
```

Comment:
```suggestion
            "@php vendor/bin/contao-setup"
```
Any reason you wanna add `--ansi` here as it's not the default that is set in the newer versions?

Reply:
Because the deprecation notice includes it. I can also remove it